qiskit/algorithms/time_evolvers/pvqd/utils.py

```diff
@@ -93,8 +85,17 @@ def evaluate_observables(theta: np.ndarray) -> Union[float, List[float]]:
         Returns:
             The observables evaluated at the ansatz parameters.
         """
-        value_dict = dict(zip(ansatz_parameters, theta))
-        sampled = sampler.convert(converted, params=value_dict)
-        return sampled.eval()
+        if isinstance(observables, list):
+            num_observables = len(observables)
+            obs = observables
+        else:
+            num_observables = 1
+            obs = [observables]
+        states = [ansatz] * num_observables
+        parameter_values = [theta] * num_observables
+
+        estimator_job = estimator.run(states, obs, parameter_values=parameter_values)
+
+        return estimator_job.result().values
```

Comment:
Do we want to wrap this, like we have done elsewhere with a try/catch and re-raise an AlgorithmError.

I do note your brought over the eval_observables to this PR but this here seems to be doing its own thing and I don't see any use of it at all in PVQD presently

Reply:
Thanks, I missed not having the catch statement here!